tools/awslint/lib/rules/common.ts

```diff
@@ -1 +1,5 @@
 export const CORE_MODULE = '@aws-cdk/cdk';
+
+export const enum TypeFqn {
+    Token = '@aws-cdk/cdk.Token'
```

Comment:
Do you mind moving all FQNs to here? they are spread across awslint...

Reply:
no problem - will pick them all up